src/github/client/github-user-client.ts

```diff
@@ -42,10 +44,30 @@ export class GitHubUserClient extends GitHubClient {
 		);
 	}
 
+	private headerConfig() {
+		return {
+			Accept: GITHUB_ACCEPT_HEADER,
+			Authorization: `token ${this.userToken}`
+		};
+	}
+
 	public async getUser(): Promise<AxiosResponse<Octokit.UsersGetAuthenticatedResponse>> {
 		return await this.get<Octokit.UsersGetAuthenticatedResponse>("/user");
 	}
 
+	public async getUserRepositories(per_page = 20, cursor?: string): Promise<GetRepositoriesResponse> {
+		try {
+			const response = await this.graphql<GetRepositoriesResponse>(GetRepositoriesQuery, {
+				per_page,
+				cursor
+			});
+			return response.data.data;
+		} catch (err) {
+			err.isRetryable = true;
```

Comment:
You can drop the isRetryable, its used in the installation/backfill code.

Reply:
Removed